### PR TITLE
Use ngettext for plural strings

### PR DESCRIPTION
**Pull request by khagaroth**
Use ngettext plural support to allow proper translations.

### CODE REVIEW

File: src/daemon/settings.vala [hunk 1]
```diff
@@ -341,11 +341,7 @@ namespace Budgie {
...
 
 				if (enabled && (time > 0)) {
-					var duration = _("Will turn off in a minute");
-
-					if (time > 1) { // use plural
-						duration = _("Will turn off in %d minutes").printf(time);
-					}
+					var duration = ngettext("Will turn off in a minute", "Will turn off in %d minutes", time).printf(time);
 
 					body = duration;
```
Comment (JoshStrobl) [inline src/daemon/settings.vala:346]: Can just have body = ngettext(...)

File: src/daemon/notifications/dbus.vala [hunk 2]
```diff
@@ -569,11 +569,7 @@
 			var summary = _("Unread Notifications");
 
 			string? body = null;
-			if (paused_notifications == 1) {
-				body = _("You received 1 notification while an application was fullscreened.");
-			} else {
-				body = _("You received %d notifications while an application was fullscreened.".printf(this.paused_notifications));
-			}
+			body = ngettext("You received %d notification while an application was fullscreened.", "You received %d notifications while an application was fullscreened.", this.paused_notifications).printf(this.paused_notifications);
```
Comment (JoshStrobl) [inline src/daemon/notifications/dbus.vala:572]: Can just have `string body = ngettext(...)`

**Review by JoshStrobl — CHANGES_REQUESTED** [review]
Otherwise LGTM. I appreciate you taking the initiative to improve this! :heart:

File: src/daemon/notifications/dbus.vala (hunk 2 above)
Comment (serebit) [reply src/daemon/notifications/dbus.vala:572]: This line is also *really* long, and should probably be broken up into multiple lines, like so:

```vala
body = ngettext(
    "You received %d notification while an application was fullscreened.",
    "You received %d notifications while an application was fullscreened.",
    this.paused_notifications
).printf(this.paused_notifications);
```

**Comment by JoshStrobl** [comment]
Went ahead and fixed my own PR items. WIll merge when CI is done.

**Pull request merged**